recipes/mold/1.3.1/conanfile.py

```diff
@@ -64,6 +64,9 @@ def _patch_sources(self):
         files.replace_in_file(self, "source_subfolder/Makefile", "MOLD_LDFLAGS += -lmimalloc", "MOLD_LDFLAGS += -L{} -lmimalloc".format(
             self.deps_cpp_info["mimalloc"].lib_paths[0]))
 
+    def build_requirements(self):
+        self.build_requires("cmake/3.24.1")
```

Comment:
Why do we need _latest_ CMake?

Reply:
To keep it longer up-to-date. Is there any reason why I shouldn't use the latest version?

Reply:
No, not an issue. I just wanted to know why the default (the one installed in the CI) was not enough. 

If it is needed only for new version `1.4.2`, please add it only to its corresponding `conanfile.py`, not to old versions.

Reply:
Build #2 failed because of cmake requrement
mold/1.4.2: 
CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
  CMake 3.18 or higher is required.  You are running version 3.15.7

Reply:
It seems, cmake shall got a version bump first... I hoped, I will get a new mold version fast

Reply:
1.3.1 continues to build with make, you don't need cmake here

Reply:
I have separated versions 1.3.x and 1.4.x one is using make another is using cmake

Reply:
Oh, I realized, you're right, it's not required for 1.3.x. Fixed now